src/github.ts

```diff
@@ -328,4 +329,39 @@ export class GitHub {
     core.info(`Writing summary`);
     await sum.addSeparator().write();
   }
+
+  public static async annotateBuildWarnings(source: string, warnings?: Array<VertexWarning>): Promise<void> {
+    (warnings ?? []).forEach(warning => {
+      if (!warning.detail || !warning.short) {
+        return;
+      }
+      const title = warning.detail.map(encoded => atob(encoded)).join(' ');
+      let message = atob(warning.short).replace(/\s\(line \d+\)$/, '');
+      if (warning.url) {
+        // https://github.com/docker/buildx/blob/d8c9ebde1fdcf659f1fa3efa6ccc27a28b0f1564/commands/build.go#L854
+        message += `\nMore info: ${warning.url}`;
```

Comment:
Wonder if this link can be made clickable by adding some markdown around it.

Reply:
Opened https://github.com/orgs/community/discussions/129897